monkey/tests/unit_tests/monkey_island/cc/services/test_authentication_service.py

```diff
@@ -39,56 +40,77 @@ def mock_repository_encryptor():
     return MagicMock(spec=ILockableEncryptor)
 
 
-def test_needs_registration__true(tmp_path, mock_repository_encryptor):
+@pytest.fixture
+def mock_island_event_queue():
+    return MagicMock(spec=IIslandEventQueue)
```

Comment:
Just use `autouse` for this and the fixture above. There's no need to have to pass them to every test.

Reply:
`autouse` doesn't work when the fixture returns an object. https://stackoverflow.com/a/37046403